src/values/SymbolValue.js

```diff
@@ -9,11 +9,11 @@
 
 /* @flow */
 
-import { PrimitiveValue } from "./index.js";
+import { PrimitiveValue, AbstractValue } from "./index.js";
 import type { Realm } from "../realm.js";
 
 export default class SymbolValue extends PrimitiveValue {
-  constructor(realm: Realm, desc?: string, intrinsicName?: string) {
+  constructor(realm: Realm, desc?: string | AbstractValue, intrinsicName?: string) {
```

Comment:
Okay, `!{value}.mightNotBeString() || !{value}.mightNotBeUndefined()`.

`mightNotBeUndefined` does not yet exist, but should be added at this occasion.